qa-tests-backend/src/test/groovy/DeclarativeConfigTest.groovy

```diff
@@ -817,24 +811,23 @@ splunk:
     // verifyDeclarativeGroups will verify that the expected groups exist within the API and share the same
     // values.
     // The number of groups found within the list of expected groups will be returned.
-    private int verifyDeclarativeGroups(String authProviderID, List<Group> expectedGroups) {
+    private verifyDeclarativeGroups(String authProviderID, List<Group> expectedGroups) {
```

Comment:
How about this:

https://github.com/stackrox/stackrox/pull/9011/files#diff-58186550e713b98873ccdebbbedf0935c31cc33c472b5ce531b2000798ee7defR320